### PR TITLE
DM-20072: Transfer author support tools from overview paper

### PR DESCRIPTION
**Pull request by timj**
In this pull request we copy the relevant files from lsst-pst/overview_paper#57

This is a resubmission of #171 with the correct ticketed branch name.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,10 @@ matrix:
     - python: "3.5"
       # Enable docker and documentation deployments
       env: LTD_MASON_BUILD=true DEPLOY_DOCKER_IMAGE=true
+    - python: '3.6'
```
Comment (womullan) [inline .travis.yml]: was it really necessary to force me to upgrade python .. ..:)

Comment (timj) [reply .travis.yml]: This is the python to use to run flake8 inside Travis. DM only use 3.7 now so I haven't tried db2authors with an older python.

**Comment by timj** [comment]
I won't merge until I get some feedback from @ivezic that he's okay with commissioning papers and the overview paper depending on lsst-texmf.

**Comment by ivezic** [comment]
I am ok with your excellent plan!

**Pull request merged**